source/1_engine/rex_engine/include/rex_engine/filesystem/path.h

```diff
@@ -0,0 +1,231 @@
+#pragma once
```

Comment:
Dunno where to put this so I will post it here. A nice addition to this would be a DirectoryIterator ( maybe this is something for a directory.h file if this exists ) I have an example already in place ( which ofc needs a rework ) but gets the idea across.
[Directory.txt](https://github.com/New-Creta/Rex/files/13742332/Directory.txt)

Reply:
that's a seperate system